concourse/pipeline.yml

```diff
@@ -210,6 +245,12 @@ jobs:
           SECRET_KEY_BASE: ((secret-key-base-production))
           WEBSITE_ROOT: https://www.gov.uk
           WORKER_INSTANCES: 20
+        on_success:
+          try:
+            put: grafana-annotate-deploy
```

Comment:
Eventualy I'd be interested in extending these into something that perhaps links to the concourse task / github PR or commit perhaps.
Eg see the pretty links here under "Add annotation after deployment": https://frederic-hemberger.de/notes/grafana/annotate-dashboards-with-deployments/#add-annotation-after-deployment

but... perhaps best to start simple and check this doesn't make a mess of our graphs first.